Updated invalid URL in log message

friek opened the pull request:
In case of unsupported 3p cookies a warning is logged which includes an invalid URL. The message refers to https://www.keycloak.org/docs/latest/securing_apps/#_modern_browsers while it should be https://www.keycloak.org/securing-apps/javascript-adapter#_modern_browsers

Closes #36168

<!---
Please read https://github.com/keycloak/keycloak/blob/main/CONTRIBUTING.md and follow these guidelines when contributing to Keycloak
-->

jonkoops commented:
@friek can you update your PR so:

1. The commit [is signed-off](https://github.com/keycloak/keycloak/pull/36169/checks?check_run_id=34989519767) to verify your code changes [comply with our license](https://github.com/keycloak/keycloak/blob/main/CONTRIBUTING.md#developers-certificate-of-origin).
2. There is descriptive title for the commit and PR that **does not** reference an issue number
3. There is a keyword in the PR and commit description to [link it to the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) (e.g. `Closes #36168`).

Thanks!

friek committed:
Updated invalid URL in log message

In case of unsupported 3p cookies a warning is logged which includes an invalid URL.
The message refers to https://www.keycloak.org/docs/latest/securing_apps/#_modern_browsers
while it should be https://www.keycloak.org/securing-apps/javascript-adapter#_modern_browsers

Closes #36168

Signed-off-by: Johan Mulder <johan@mumasoft.nl>

friek commented:
Here you go. Is this sufficient?

jonkoops reviewed (approved):
Looks great, thanks for your contribution!
